src/main/scala/com/fulcrumgenomics/illumina/RunInfo.scala

```diff
@@ -50,7 +50,11 @@ case class RunInfo
   run_date: Iso8601Date,
   read_structure: ReadStructure,
   num_lanes: Int
-) extends Metric
+) extends Metric {
+  override protected def formatValue(value: Any): String = {
+    if (value.isInstanceOf[ReadStructure]) value.toString else super.formatValue(value)
+  }
+}
```

Comment:
How did this change become necessary?

Reply:
`ReadStructure` is a collection type, so the by default, the read segments will be comma-delimited (e.g. `8M,100T`) whereas we just want it represented as a string (e.g. `8M100T`).